packages/eslint-config-react/package.json

```diff
@@ -12,13 +12,10 @@
     "eslintconfig"
   ],
   "peerDependencies": {
-    "eslint": ">=7",
-    "eslint-plugin-import": ">=2.22.0",
-    "eslint-plugin-jsx-a11y": ">=6.4.0",
-    "eslint-plugin-react": ">=7.23",
     "typescript": ">=4.2"
   },
   "dependencies": {
+    "@rushstack/eslint-patch": "^1.1.0",
```

Comment:
Those comments apply to the other `package.json` as well:
- The dependencies that we removed from `devDependencies` should now be put in `dependencies` to work correctly in installed projects.
- Double check if `eslint` should be installed as a dependency or as a devDependency + peerDependency. Maybe if we don't find anything for that just consider this as a eslint plugin and install it as a dependency, we'll see afterwards if it works or not in the starter project.

Just in other package.json (there are still peerDependencies related to ESLint plugins.